### PR TITLE
Snap script cleanup

### PR DESCRIPTION
**Pull request by tonyespy**
This commit cleans up warnings produced by shellcheck when run against the snap start & stop scripts.

### CODE REVIEW

**Comment by tsconn23** [comment]
@tonyespy Sign-off is missing. Is this a replacement for #331 ? It appears to touch the same files.

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -90,6 +90,11 @@ parts:
       gunzip "./$PACKAGE.gz"
       tar xvf "./$PACKAGE" --strip-components=1
       popd
+    stage:
```
Comment (anonymouse64) [inline snap/snapcraft.yaml]: Note: these changes are from the same commits as in https://github.com/edgexfoundry/edgex-go/pull/331
Not a blocker, just pointing this out

**Comment by tonyespy** [comment]
That's because I stacked it on top of the previous PR.

**Comment by JPWKU** [comment]
recheck

**Comment by JPWKU** [comment]
@tonyespy There are some conflicts that need to be resolved first.

**Comment by tsconn23** [comment]
@anonymouse64 Some merge conflicts need to be resolved before we can pull this in.

**Pull request merged**